chromium_src/components/password_manager/core/common/password_manager_features.cc

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "src/components/password_manager/core/common/password_manager_features.cc"
+
+#include "base/feature_override.h"
+#include "build/build_config.h"
+
+namespace password_manager::features {
+
+#if BUILDFLAG(IS_ANDROID)
+OVERRIDE_FEATURE_DEFAULT_STATES({{
+    {kUnifiedPasswordManagerLocalPasswordsMigrationWarning,
```

Comment:
Please, also add to `IN_PROC_BROWSER_TEST_F(BraveMainDelegateBrowserTest, DisabledFeatures)` in https://github.com/brave/brave-core/blob/master/app/brave_main_delegate_browsertest.cc

Reply:
yeah, we have old browser tests that were done for desktop only unfortunately, let's create a follow up and merge that PR

Reply:
the follow-up issue is https://github.com/brave/brave-browser/issues/36533